source/index.md

```diff
@@ -38,12 +38,15 @@ use:
         <h1 class="box-headline">Trusted by</h1>
         <a href="https://www.atmail.com" title="sabre/dav is trusted by atmail.">
             <img src="{{ site.url }}/img/trusted/atmail.png" alt="atmail works with sabre/dav">
-        </a><a href="http://tech.blog.box.com/2014/10/in-search-of-an-open-source-webdav-solution/" title="sabre/dav is trusted by Box.">
```

Comment:
Think we could drop these links? I feel that they are a little bit unneeded. They are already on the homepage as well, lets not make the advertising overly obvious. I think it's a bit much.